tickets/src/main/java/no/unit/nva/publication/ticket/update/updateTicketAssigneeHandler.java

```diff
@@ -0,0 +1,139 @@
+import no.unit.nva.model.Publication;
+import no.unit.nva.publication.model.business.DoiRequest;
+import no.unit.nva.publication.model.business.TicketEntry;
+import no.unit.nva.publication.model.business.UserInstance;
+import no.unit.nva.publication.service.impl.ResourceService;
+import no.unit.nva.publication.service.impl.TicketService;
+import no.unit.nva.publication.ticket.TicketDto;
+import no.unit.nva.publication.ticket.TicketHandler;
+import nva.commons.apigateway.AccessRight;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadGatewayException;
+import nva.commons.apigateway.exceptions.BadMethodException;
+import nva.commons.apigateway.exceptions.ForbiddenException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.apigateway.exceptions.UnauthorizedException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.secrets.SecretsReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class updateTicketAssigneeHandler extends TicketHandler<TicketDto, Void> {
+
+    private final TicketService ticketService;
+    private final ResourceService resourceService;
+    private final DoiClient doiClient;
+    private static final Logger logger = LoggerFactory.getLogger(UpdateTicketStatusHandler.class);
+    public static final String COULD_NOT_CREATE_FINDABLE_DOI = "Could not create findable doi";
+    public static final String PUBLICATION_WITH_IDENTIFIER_S_DOES_NOT_SATISFY_DOI_REQUIREMENTS
+        = "Publication with identifier  %s, does not satisfy DOI requirements";
+
+    @JacocoGenerated
+    public updateTicketAssigneeHandler() {
+        this(TicketService.defaultService(),
+             ResourceService.defaultService(), new DataCiteDoiClient(HttpClient.newHttpClient(),
+                                                                     SecretsReader.defaultSecretsManagerClient(),
+                                                                     new Environment().readEnv("API_HOST")));
+    }
+
+    public updateTicketAssigneeHandler(TicketService ticketService, ResourceService resourceService,
+                                       DoiClient doiClient) {
+        super(TicketDto.class);
+        this.ticketService = ticketService;
+        this.resourceService = resourceService;
+        this.doiClient = doiClient;
+    }
+
+    @Override
+    protected Void processInput(TicketDto input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        var ticketIdentifier = extractTicketIdentifierFromPath(requestInfo);
+        var ticket = ticketService.fetchTicketByIdentifier(ticketIdentifier);
+
+        if (userIsNotAuthorized(requestInfo, ticket)) {
+            throw new ForbiddenException();
+        }
+        if (ticket instanceof DoiRequest) {
+            doiTicketSideEffects(input, requestInfo);
+        }
```

Comment:
Not sure if you have to include doi side effects here. When updating status for pubishingRequest, we have side effects to make doi findable/delete it. In your case you are adding assignee, that should not do anything with doi (as I see it)